vitamins/io.github.spacingbat3.webcord.desktop

```
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=WebCord
+Exec=run.sh
+Terminal=false
+Type=Application
+Icon=io.github.spacingbat3.webcord
+StartupWMClass=WebCord
```

We should add a "Discord" and "Fosscord" keyword so typing "Discord" or "Fosscord" in the app menu shows WebCord.
```suggestion
StartupWMClass=WebCord
Keywords=Discord;Fosscord;
```

---

I just tested on my end. It works.